src/main/java/games/strategy/triplea/delegate/MoveValidator.java

```diff
@@ -308,7 +308,7 @@ private static MoveValidationResult validateCombat(final GameData data, final Co
     }
     // if there is a neutral in the middle must stop unless all are air or getNeutralsBlitzable
     if (route.hasNeutralBeforeEnd()) {
-      if (!Match.allMatchNotEmpty(units, Matches.UnitIsAir) && !isNeutralsBlitzable(data)) {
+      if (units.isEmpty() || !Match.allMatch(units, Matches.UnitIsAir) && !isNeutralsBlitzable(data)) {
```

Comment:
Do we need parentheses here due to `&&` having a higher precedence than `||`?  For example:

```java
(units.isEmpty() || !Match.allMatch(units, Matches.UnitIsAir)) && !isNeutralsBlitzable(data)
```